src/_lib/tzTokenizeDate/test.js

```diff
@@ -20,6 +20,11 @@ describe('tzTokenizeDate', function () {
     assert.deepEqual(result, [2020, 1, 23, 0, 0, 0])
   })
 
+  it('works with year < 100', function () {
+    var result = tzTokenizeDate(new Date('0099-01-01T00:00:00.000Z'), 'UTC')
+    assert.deepEqual(result, [99, 1, 1, 0, 0, 0, 0])
```

Comment:
Oops the last zero needs to be gone for now, please review changes first, I'll force push later.

Reply:
Done!